src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import CloseIcon from "@mui/icons-material/Close";
+import { useDispatch, useSelector } from "react-redux";
+import UndoIcon from "@mui/icons-material/Undo";
+import DeleteIcon from "@mui/icons-material/Delete";
+import GoBackComponent from "../../../../goBackButton/GoBackButton";
+import {
+  changeSelectedPropertyForEdit,
+  changeToggleDeletedDialog,
+  changeToggleDeletedProperties,
+  deleteProperty,
+} from "../../../../../features/tableDataInfo/tableDataInfoSlice";
+import {
+  changeShowCreateTabPopover,
+  changeShowView,
+  changeToggleAddPropertyPopover,
+} from "../../../../../features/showPopoversInfo/showPopoverInfoSlice";
+import style from "./properties.module.scss";
+import propertyIcons from "../../../../propertyIcons/propertyIcons";
+import DeletedDialog from "../../../../deletedDialog/DeletedDialog";
+
+export default function DeletedProperties() {
+  const dispatch = useDispatch();
+
+  const { data, toggleDeletedDialog } = useSelector(
+    (store) => store.tableDataInfo
+  );
+
+  const onCloseBtn = () => {
+    dispatch(changeShowView(false));
+    dispatch(changeShowCreateTabPopover(false));
+    dispatch(changeToggleDeletedProperties());
+  };
+
+  const unDeleteProperty = (id) => {
+    dispatch(changeSelectedPropertyForEdit(id));
+    dispatch(changeToggleDeletedProperties());
+    dispatch(changeToggleAddPropertyPopover(true));
+  };
```

Comment:
are you sure that you need all these global states?

Reply:
yes I use them in global